automation/devops_automation_infra/utils/postgresql.py

```diff
@@ -10,4 +10,23 @@ def truncate_all(connection):
 
     sql.execute(connection, queries)
     connection.rollback()
-    connection.reset()
\ No newline at end of file
+    connection.reset()
+
+
+def group_exists_in_pg(connection, group_id):
+    response = sql.fetchone(connection,
+                            f"SELECT * FROM camera_groups WHERE id = '{group_id}' AND \"isDeleted\" = False")
```

Comment:
why in devops automation infra ? this is supposed to be in camera service

Reply:
camera service util